packages/server/start/mpeSocket.ts

```diff
@@ -85,4 +85,54 @@ export default function initMpeSocketEventListeners(socket: TypedSocket): void {
             });
         }
     });
+
+    socket.on('MPE_CHANGE_TRACK_ORDER_DOWN', async (raw) => {
+        try {
+            const { fromIndex, trackID, roomID } =
+                MpeRoomClientToServerChangeTrackOrderUpDownArgs.parse(raw);
+
+            const { user, deviceID } =
+                await SocketLifecycle.getSocketConnectionCredentials(socket);
+
+            await MpeRoomsWsController.onChangeTrackOrder(
+                {
+                    roomID,
+                    trackID,
+                    userID: user.uuid,
+                    deviceID,
+                    fromIndex,
+                    operationToApply:
+                        MpeChangeTrackOrderOperationToApply.Values.DOWN,
```

Comment:
It seems that you prefer enums over literal strings unions 👀

Reply:
No big deal to me :p
Just been doing the same as we've done for the `MtvPlayingModes` ( it was also me lol )